docs/source/howto/upgrading-v5.md

```diff
@@ -95,18 +93,18 @@ Prior to JupyterHub 5, JupyterHub Authenticators had the _implicit_ default beha
 This behavior was considered a too-permissive default in Authenticators that source large user pools like OAuthenticator, which would accept e.g. all users with a Google account by default.
 As a result, OAuthenticator 16 introduced two configuration options:
 
-1. `Authenticator.allow_all` (default: False)
-2. `Authenticator.allow_existing_users` (default: True if allowed_users is non-empty, False otherwise)
+1. `OAuthenticator.allow_all` (default: False)
+2. `OAuthenticator.allow_existing_users` (default: True if allowed_users is non-empty, False otherwise)
```

Comment:
For OAuthenticator, this is defaulting to `False` actually :p

Reply:
reordered to clarify that these describe JupyterHub defaults, not OAuthenticator